app/views/teacher_training_adviser/steps/_identity.html.erb

```diff
@@ -18,7 +18,7 @@
 </p>
 
 <p>
-  If you're already a qualified teacher, you'll need English qualified teacher status (QTS) and want to
+  If you're already a qualified teacher, you'll need English qualified teacher status (QTS), previous experience of teaching in the UK or have trained to teach in the UK, and want to
```

Comment:
```suggestion
If you're already a qualified teacher, you'll need English qualified teacher status (QTS), previous experience of teaching in the UK or to have trained to teach in the UK, and want to teach in an English state school.
```

Think it might need a `to` in the trained bit?

Also wonder if we could lose the last bit to make it shorter - returners shouldn't be coming to this page really (we need to look at the whole thing really!) - but do disagree if needed

NB my suggestion covers line 22 as well!